Mimir.Worker/Poller/TxPoller.cs

```diff
@@ -73,6 +74,7 @@ MongoDbService dbService
             CollectionNames.GetCollectionName<RaiderStateDocument>(),
             CollectionNames.GetCollectionName<WorldBossStateDocument>(),
             CollectionNames.GetCollectionName<WorldBossKillRewardRecordDocument>(),
+            CollectionNames.GetCollectionName<WorldBossKillRewardRecordDocument>(),
```

Comment:
It seems duplicated with the above line, is it necessary?

Reply:
@.@ it seems my mistake thanks